include/tvm/tir/builtin.h

```diff
@@ -343,6 +343,16 @@ TVM_DLL const Op& tvm_stack_make_array();
  */
 TVM_DLL const Op& tvm_call_packed();
 
+/*!
+ * \brief See pesudo code
+ *
+ *  int tvm_call_packed(fname, TVMValue* args) {
+ *     (*fname)(args, type_code_of(args), len(args));
+ *     return 0;
+ *  }
+ */
+TVM_DLL const Op& tvm_call_cpacked();
```

Comment:
https://github.com/apache/tvm/pull/7932 contains some updated clarifications that we might want to incorporate here

Reply:
in particular, we might also want to support POD return types

Reply:
Hi @tqchen, do you mean updating the doc string? If the overall PR is fine, I would stick with this for now and update the comments in #7932 . If this last PR gets merged before AOT, then I would update the comment. What do you think?

Reply:
Yah I meant docstring. A separate update PR is also good

Reply:
Given #7932 has been merged, I updated the docstring